modules/data-entry/src/main/frontend/src/dataHomepage/FormView.jsx

```diff
@@ -75,7 +76,7 @@ function FormView(props) {
   ]
 
   const tabs = ["Completed", "Draft"];
-  const activeTabParam = location.hash.substr(1);
+  const activeTabParam = location.hash.substr(1).split('&')[0];
```

Comment:
We should refactor how we identify the active tab as well:

```suggestion
  const activeTabParam = new URLSearchParams(window.location.hash.substring(1)).get("forms:activeTab");
```

Of course, it's also needed to update the part where the URL is generated.